schema/method_units/physical/psp/set.json

```diff
@@ -0,0 +1,61 @@
+{
+    "schemaId": "method-physical-psp-set",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "Schema for a set of pseudopotential files",
+    "properties": {
+        "type": {
+            "description": "type of pseudization",
+            "enum": [
+                "paw",
+                "us",
+                "nc"
+            ],
+            "type": "string"
+        },
+        "elements": {
+            "description": "List of elements in pseudopotential set",
+            "$ref": "../../../core/primitive/array_of_strings.json"
+        },
+        "modelReference": {
+            "description": "Unique identifier of the model used to generate the PP set",
+            "type": "string"
+        },
+        "format": {
+            "description": "type of pseudopotential format",
+            "enum": [
+                "upf",
+                "vasp",
+                "paw"
+            ],
+            "type": "string"
+        },
+        "name": {
+            "description": "Name for pseudopotential set",
+            "type": "string"
+        },
+        "source": {
+            "description": "explains the source of where this came from",
+            "$ref": "file/origin.json#/definitions/ppSources"
+        },
+
+        "version": {
+            "description": "explains the version of where this came from",
+            "type": "string"
+        },
+        "files": {
+            "description": "pseudopotentials used inside this set",
+            "type": "array",
+            "items": {
+                "$ref": "file.json"
+            }
+        }
+    },
+    "required": [
+        "type",
+        "elements",
+        "modelReference",
+        "format",
+        "name",
+        "files"
+    ]
+}
```

Comment:
This will need more clarification. Let's put a task into the epic for the pseudopotential sets / workflow datasets and link to this file/PR to save the work. Then remove the file for now.